blockstore/blockstore.go

```diff
@@ -64,6 +65,13 @@ type Blockstore interface {
 	HashOnRead(enabled bool)
 }
 
+// TxnBlockstore is a blockstore interface that supports GetMany and PutMany methods using ds.TxnDatastore
+type TxnBlockstore interface {
```

Comment:
I think `TxnBlockstore` is wrong because the blockstore does not provide transactions, `GetManyBlockstore` was fine mb.